workers/unity/Packages/io.improbable.gdk.core/Worker/ConnectionHandlers/ConnectionFlows.cs

```diff
@@ -55,6 +60,8 @@ public ReceptionistFlow(string workerId, IConnectionFlowInitializer<Receptionist
 
         public async Task<Connection> CreateAsync(ConnectionParameters parameters, CancellationToken? token = null)
         {
+            parameters.Network.UseExternalIp = UseExternalIp;
```

Comment:
Just noticed that if we set `UseExternalIp` outside of the connection flow, it gets overwritten by whatever we set as the default

e.g. in the editor if we set the `UseExternalIp` field on the `ClientConnector`, it gets overwritten when we call `CreateAsync`

Reply:
Should I handle this only on the `Receptionist` or on the `Locator` as well? I could set the cli arg to take precedence; i.e.  if we set `UseExternalIp` on in the connection flow we'll use this value, if not just use whatever's supplied through the `ConnectionParameters`.

Reply:
We won't have to worry about what flow gets used if we move this into `CommandLineConnectionParameterInitializer`